examples/securenn/network_b.py

```diff
@@ -99,25 +74,37 @@ def build_training_graph(self, training_data) -> List[tf.Tensor]:
 
     Returns a list of the trained model's parameters.
     """
-    # model parameters and initial values
-    wconv1 = weight_variable([self.KERNEL,
-                              self.KERNEL,
-                              self.IN_CHANNELS,
-                              self.HIDDEN_CHANNELS], 1.)
-    bconv1 = bias_variable([1, 1, self.HIDDEN_CHANNELS])
-    wconv2 = weight_variable([self.KERNEL,
-                              self.KERNEL,
-                              self.HIDDEN_CHANNELS,
-                              self.HIDDEN_CHANNELS], 1.)
-    bconv2 = bias_variable([1, 1, self.HIDDEN_CHANNELS])
-    wfc1 = weight_variable([self.HIDDEN_FC1, self.HIDDEN_FC2], 1.)
-    bfc1 = bias_variable([self.HIDDEN_FC2])
-    wfc2 = weight_variable([self.HIDDEN_FC2, self.OUT_N], 1.)
-    bfc2 = bias_variable([self.OUT_N])
-    params = [wconv1, bconv1, wconv2, bconv2, wfc1, bfc1, wfc2, bfc2]
+    model = keras.Sequential()
+    model.add(keras.layers.Conv2D(self.HIDDEN_CHANNELS,
+                                  (self.KERNEL, self.KERNEL),
+                                  batch_input_shape=(self.BATCH_SIZE,
+                                                     self.IN_DIM,
+                                                     self.IN_DIM,
+                                                     self.IN_CHANNELS)))
+    model.add(keras.layers.Activation('relu'))
+    model.add(keras.layers.AveragePooling2D())
+    model.add(keras.layers.Conv2D(
+        self.HIDDEN_CHANNELS, (self.KERNEL, self.KERNEL)))
```

Comment:
Let's keep the indentation consistent for the `Conv2d`. You can use the same approach as Con2d above.